src/main/java/com/training/web/pages/rccl/HomePage.java

```diff
@@ -11,6 +11,7 @@
     private String upComingCruises = "//span[text()='Upcoming cruises']";
     private String addCruise = "//button[text()='Add a cruise']";
     private String planCruise = "//a[text()='Plan a new cruise']";
+    private String idText="//span[@class='loyalty-meter__item-title']";
```

Comment:
make the naming more specific, its the loyalty id. If anyone tries to use this code - it will get confused with the objects.